packages/create-workleap/index.ts

```diff
@@ -0,0 +1,109 @@
+import * as child_process from "child_process";
+import process from "process";
+
+import { Intro, Select, Note, Text, Outro } from "./prompts";
+import * as pkg from "./package.json";
+import { StartSpinner, StopSpinner } from "./spinner";
```

Comment:
Is there any particular reason why those spinners should be in a distinct files? Are they re-used somewhere?